tarantool/mesh_connection.py

```diff
@@ -63,3 +131,39 @@ def _opt_reconnect(self):
                 self.port = addr['port']
         else:
             raise NetworkError
+
+        if self.authenticated and self.get_nodes_function_name:
+            now = time.time()
+            if now - self.last_nodes_refresh > self.nodes_refresh_interval:
+                self.refresh_nodes(now)
+
+    def refresh_nodes(self, cur_time):
+        '''
+        Refreshes nodes list by calling Lua function with name
+        self.get_nodes_function_name on the current node. If this field is None
+        no refresh occurs. Usually you don't need to call this function manually
+        since it's called automatically during reconnect every
+        self.nodes_refresh_interval seconds.
+        '''
+        resp = super(MeshConnection, self).call_ex(self.get_nodes_function_name,
+                                                   [], reconnect=False)
+
+        if not (resp.data and resp.data[0]):
+            return
+
+        addrs_raw = resp.data[0]
+        if type(addrs_raw) is list:
```

Comment:
YOu can use `is not` and `return` here to decrease indent level count of the block.

Reply:
Cleaned